src/test/unit/blackbox_unittest.cc

```diff
@@ -199,32 +202,10 @@ TEST(BlackboxTest, Test_8kHz)
     EXPECT_TRUE(blackboxShouldLogPFrame());
 }
 
-TEST(BlackboxTest, Test_zero_p_ratio)
```

Comment:
How about, instead of removing this test, you rewrite it to check the `blackboxPInterval > blackboxIInterval` case that should have the same acceptance criteria (i.e. no P frames logged)?